README.md

```diff
@@ -18,13 +18,26 @@ YourApp/<br>
 --**ROOT**<br>
 --node_modules/<br>
 --utils/<br>
-----**NOT ROOT**<br>
-----coolUtil.js<br>
+  --**NOT ROOT**<br>
+  --coolUtil.js<br>
 --server/<br>
-----server.js<br>
+  --server.js<br>
 --index.js<br>
---package.json<br>
---README.md<br>
+--package.json<br>                              
+--README.md<br>                                  
+                                        __________
+ ######                                 \          \
+######## --------------------------------           |
+ ######                                             / _____________
+ ######                                             \ -------------\\
+######## --------------------------------           |               \\
+ ######                                 \__________/                 \\
+                                                                    /   \
+                                                                  /       \
+                                                                /           \
+                                                      _________/             \_________
+                                                     /                                 \
+                                                    /                                   \
```

Comment:
I don't believe this is the 'Hog' you were looking for.

<img width="958" alt="screen shot 2016-05-08 at 3 31 14 pm" src="https://cloud.githubusercontent.com/assets/9343128/15100775/0df230dc-1532-11e6-8b20-ceb4cbaca664.png">